util/internal/integrationtest/mapper/orm/src/main/java/org/hibernate/search/util/impl/integrationtest/mapper/orm/multitenancy/impl/MultitenancyTestHelper.java

```diff
@@ -39,7 +38,6 @@ private void attachTo(SimpleSessionFactoryBuilder builder) {
 		builder.onServiceRegistryBuilder( srb -> srb.addInitiator(
 				new MultitenancyTestHelperSchemaManagementTool.Initiator( tenantIds ) ) );
 
-		builder.setProperty( AvailableSettings.MULTI_TENANT, MultiTenancyStrategy.DATABASE.name() );
```

Comment:
I'll split this to a separate commit, since this is the only part of your commit that cannot work on ORM 5.6.